app/javascript/shared/components/tags.jsx

```diff
@@ -34,7 +34,7 @@ class Tags extends Component {
     // 250ms without invoking the function at the leading edge of the timeout
     // NOTE: this seems the best combination of wait time and options to avoid
     // flickering and text replacement during autocomplete
-    this.debouncedTagSearch = debounce(this.handleInput.bind(this), 250);
+    this.debouncedTagSearch = debounceAction(this.handleInput.bind(this), 250);
```

Comment:
I think this is going to have `{leading: true}` added even if we shouldn't use it